products/opensuse/main.pm

```diff
@@ -38,6 +38,10 @@ sub is_leap {
     return get_var('VERSION') =~ /^42:S/;
 }
 
+sub is_leap_42_3_or_newer {
+    return is_leap && check_var('VERSION', '42.3');
+}
+
```

Comment:
@mnowaksuse The problem with method is that I'm not sure how next version will be called, as otherwise we could simply compare float numbers.
Regarding main_common, should I then move  is_leap and is_tumbleweed there as well? My understanding is that these are opensuse specific. Maybe lib/utils then, as we have similar functions for sle there?